packages/event-target-impl/src/index.node.ts

```diff
@@ -0,0 +1,15 @@
+import { setMaxListeners } from 'node:events';
+
+export const AbortController = class extends globalThis.AbortController {
+    constructor(...args: ConstructorParameters<typeof globalThis.AbortController>) {
+        super(...args);
+        setMaxListeners(Number.MAX_SAFE_INTEGER, this.signal);
+    }
+};
```

Comment:
What's the reason behind setting this inside the `AbortController` constructor as well?

Reply:
`signal` is an `EventTarget`, and apparently attaching `addEventListener('abort', ...)` to it more than 10 times also raises this warning.